plugin/lsp.vim

```diff
@@ -65,10 +65,10 @@ command! -range -nargs=* -complete=customlist,lsp#ui#vim#code_action#complete Ls
       \   'selection': <range> != 0,
       \   'query': '<args>'
       \ })
-command! LspDeclaration call lsp#ui#vim#declaration(0)
-command! LspPeekDeclaration call lsp#ui#vim#declaration(1)
-command! LspDefinition call lsp#ui#vim#definition(0)
-command! LspPeekDefinition call lsp#ui#vim#definition(1)
+command! -bang -nargs=? -complete=customlist,lsp#ui#vim#open_where LspDeclaration call lsp#ui#vim#declaration('<args>')
+command! LspPeekDeclaration call lsp#ui#vim#declaration('preview')
```

Comment:
Peek commands are kept around for backwards compatibility. They are identical to `:Lsp<Command> preview`.